boskos/cmd/cli/cli.go

```diff
@@ -240,6 +247,76 @@ Examples:
 	}
 	root.AddCommand(metrics)
 
+	heartbeat := &cobra.Command{
+		Use:   "heartbeat",
+		Short: "Send a heartbeat for a resource reservation",
+		Long: `Send a heartbeat for a resource reservation
+
+When the Boskos reaper is deployed, resource lessees must send a
+heartbeat for every lease they hold or the leases will be revoked.
+This command will send a heartbeat at the provided period and is
+blocking.
+
+Examples:
+
+  # Acquire one clean "my-thing" and mark it dirty when leasing
+  $ resource="$( boskosctl acquire --type my-thing --state clean --target-state dirty )"
+  # Send periodic heartbeat for the lease
+  $ boskosctl heartbeat --resource "${resource}" --period 30s &`,
```

Comment:
looks like the `&` is extra here? also make --timeout an example as well

Reply:
No, it's blocking so you want it to run in the background